--- jupyter_client-6.1.7/setup.py.orig	2020-08-25 22:26:18.000000000 +0200
+++ jupyter_client-6.1.7/setup.py	2020-09-14 21:38:06.018627771 +0200
@@ -77,7 +77,7 @@
     ],
     python_requires  = '>=3.5',
     extras_require   = {
-        'test': ['ipykernel', 'ipython', 'mock', 'pytest', 'pytest-asyncio', 'async_generator', 'pytest-timeout'],
+        'test': ['ipykernel', 'ipython', 'pytest', 'pytest-asyncio', 'async_generator', 'pytest-timeout'],
     },
     cmdclass         = {
         'bdist_egg': bdist_egg if 'bdist_egg' in sys.argv else bdist_egg_disabled,
